### PR TITLE
Translate the news about CVE-2015-1855 in FR.

### PR DESCRIPTION
**Pull request by GRoguelon**
Hi,

The is the translation about the last 4 news.

Thanks.

cc: @chatgris

### CODE REVIEW

File: fr/news/_posts/2015-04-13-ruby-2-2-2-released.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+date: 2015-04-13 12:00:00 +0000
+lang: fr
+---
+
+Nous avons le plaisir d'annoncer la sorti de Ruby 2.2.2.
+C'est une mise à jour mineure de la série stable 2.2.
+
+Cette version inclut un correctif de sécurite d'une extension d'OpenSSL sur la vérification du nom d'hôte.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline fr/news/_posts/2015-04-13-ruby-2-2-2-released.md]: Should be

```markdown
* [CVE-2015-1855: Vérification des nom d'hôte par Ruby OpenSSL](https://www.ruby-lang.org/fr/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

File: fr/news/_posts/2015-04-13-ruby-2-1-6-released.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+date: 2015-04-13 12:00:00 +0000
+lang: fr
+---
+
+Ruby 2.1.6 est disponible.
+
+Cette version inclut un correctif de sécurite d'une extension d'OpenSSL.
+Veuillez consulter le sujet suivant pour plus d'informations.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline fr/news/_posts/2015-04-13-ruby-2-1-6-released.md]: Should be

```markdown
* [CVE-2015-1855: Vérification des nom d'hôte par Ruby OpenSSL](https://www.ruby-lang.org/fr/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

File: fr/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+date: 2015-04-13 12:00:00 +0000
+lang: fr
+---
+
+Nous avons le plaisir d'annoncer la sorti de Ruby 2.0.0-p645.
+
+Cette version inclut un correctif de sécurite d'une extension d'OpenSSL.
+Veuillez consulter le sujet suivant pour plus d'informations.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline fr/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md]: Should be

```markdown
* [CVE-2015-1855: Vérification des nom d'hôte par Ruby OpenSSL](https://www.ruby-lang.org/fr/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

**Comment by JuanitoFatas** [comment]
:+1:

**Comment by GRoguelon** [comment]
Done! Thanks.

**Comment by JuanitoFatas** [comment]
De rien :smile:

**Comment by GRoguelon** [comment]
Hi,

Any news about the status of this merge request?

Thanks

File: fr/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+date: 2015-04-13 12:00:00 +0000
+lang: fr
+---
+
+Nous avons le plaisir d'annoncer la sorti de Ruby 2.0.0-p645.
+
+Cette version inclut un correctif de sécurite d'une extension d'OpenSSL.
+Veuillez consulter le sujet suivant pour plus d'informations.
+
+* [CVE-2015-1855: Vérification des nom d'hôte par Ruby OpenSSL](https://www.ruby-lang.org/fr/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (chatgris) [inline fr/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md]: I guess it's `noms` here ?

File: fr/news/_posts/2015-04-13-ruby-2-1-6-released.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+title: "Ruby 2.1.6 est disponible"
+author: "usa"
+translator: "Geoffrey Roguelon"
+date: 2015-04-13 12:00:00 +0000
+lang: fr
+---
+
+Ruby 2.1.6 est disponible.
+
+Cette version inclut un correctif de sécurite d'une extension d'OpenSSL.
```
Comment (chatgris) [inline fr/news/_posts/2015-04-13-ruby-2-1-6-released.md]: Small typo on `sécurité`. :ok_hand:

File: fr/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+layout: news_post
+title: "CVE-2015-1855: Vérification des nom d'hôte par Ruby OpenSSL"
+author: "zzak"
+translator: "Geoffrey Roguelon"
+tags: security
+date: 2015-04-13 12:00:00 +0000
+lang: fr
+---
+
+L'entension OpenSSL de Ruby souffre d'une faille rendant la correspondance trop permissive des noms d'hôte, cela peut mener à des bugs similaires à [CVE-2014-1492][CVE-2014-1492]. Des problèmes similaires ont été identifiés dans [Python][python-hostname-bug].
```
Comment (chatgris) [inline fr/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: `extension` :stuck_out_tongue:

File: fr/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+Cette faille a reçu l'identifiant CVE [CVE-2015-1855][CVE-2015-1855].
+
+Nous vous recommandons vivement de mettre à jour Ruby.
+
+## Détails
+
+Après relecture de la [RFC 6125][RFC-6125] et de la [RFC 5280][RFC-5280], nous avons trouvé plusieurs violations de la correspondance des noms d'hôtes et plus particulièrement avec les certficats wildcard.
+
+L'entension OpenSSL de Ruby fournira dorénavant un algorithme de correspondance par chaîne de caractères qui a un comportement _plus_ stricte, comme spécifié par ces RFCs. Plus particulièrement, la correspondance de plusieurs wildcard par sujet/SAN n'est plus autorisée. Enfin, la comparaison de ces valeurs est insensible à la casse.
+
```
Comment (chatgris) [inline fr/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: I think it is `strict` instead of `stricte`.

**Comment by JuanitoFatas** [comment]
Hi @GRoguelon, could you help to address above comments? Thank you!

**Comment by GRoguelon** [comment]
Sorry, I missed the notification, I will fix that in the next hours.

**Comment by GRoguelon** [comment]
Hi,

I fixed the fixes of @chatgris, thanks.

cc @JuanitoFatas

**Pull request merged**